src/views/SelectionInfoView.tsx

```diff
@@ -51,9 +51,35 @@ export default function SelectionInfoView(
         else angleOutput = <p>Unknown angle (missing marker)</p>;
     }
     else angleOutput = <></>; //no output: not enough selected
+    /* Always display force metadata when a force file is loaded */
```

Comment:
We could add a blank line immediately above this line to separate it from the angle output section